server/usr/local/bin/add-group

```diff
@@ -0,0 +1,80 @@
+#!/bin/ash
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+# shellcheck shell=dash disable=SC3001,SC3003,SC3028
+
+usage_error()
+{
+    exec 1>&2
+    [ -n "$1" ] && printf "Error: %s\n" "$1"
+    printf "Usage: %s [OPTIONS] GROUP\n" "$0"
+    printf "    GROUP                    Add/Create a new GROUP of users\n"
+    printf "    OPTIONS\n"
+    printf "      -u user1,user2,...    Comma-separated list of initial group members\n"
+    printf "      -d description        The group description\n"
+    printf "      -g numgid             Set the numeric group identifier\n"
+    exit 2
+}
+
+# XXX default ngid? e.g. max uidNumber + 1
```

Comment:
Why use a random GID? Maybe it's better to get current maximum GID and add 1.

Reply:
uidNumber and gidNumber should not be important for ns8 apps. I decided to postpone this issue after considering the users import script implementation and measuring performances with 10k users.

Reply:
I still do not like a random UID/GID